tasks/test.py

```diff
@@ -33,26 +34,55 @@ def ensure_bytes(s):
     return s
 
 
+@contextmanager
+def environ(env):
+    original_environ = os.environ.copy()
+    os.environ.update(env)
+    yield
+    for var in env.keys():
+        if var in original_environ:
+            os.environ[var] = original_environ[var]
+        else:
+            os.environ.pop(var)
+
+
 TOOL_LIST = [
     'github.com/client9/misspell/cmd/misspell',
     'github.com/frapposelli/wwhrd',
     'github.com/fzipp/gocyclo',
     'github.com/go-enry/go-license-detector/v4/cmd/license-detector',
-    'github.com/golangci/golangci-lint/cmd/golangci-lint',
     'github.com/gordonklaus/ineffassign',
     'github.com/goware/modvendor',
     'github.com/mgechev/revive',
+    'github.com/stormcat24/protodep',
+    'golang.org/x/lint/golint',
     'gotest.tools/gotestsum',
     'honnef.co/go/tools/cmd/staticcheck',
 ]
 
+TOOL_LIST_GET = [
```

Comment:
Why do we need this? I would like to avoid this: it goes against Go best practices, it does not let us use Dependabot to bump our tooling and it breaks the purpose of go mod tidy checks. If there is some sort of conflict with the versions on `internal/tools/go.mod` we should create a new `internal/tools/proto` module with these versions instead.

Reply:
We need this because we need those pins. Indeed I was unable to get this to work properly with `internal/tools`, I'm fine with the `internal/tools/proto` approach.